pytorch_lightning/callbacks/pruning.py

```diff
@@ -407,6 +408,23 @@ def on_train_end(self, trainer: 'pl.Trainer', pl_module: LightningModule) -> Non
             rank_zero_debug("`ModelPruning.on_train_end`. Pruning is made permanent for this checkpoint")
             self.make_pruning_permanent(pl_module)
 
+    def _make_state_dict_permanent(self, pl_module: LightningModule) -> Dict[str, Any]:
+        state_dict = pl_module.state_dict()
+
+        # find the mask and the original weights.
+        map_pruned_params = {k.replace("_mask", "") for k in state_dict.keys() if k.endswith("_mask")}
```

Comment:
Don't we have access to the list of parameters to prune with `self._parameters_to_prune`?